Dockerfile

```diff
@@ -1,24 +1,35 @@
-FROM node:6-stretch
+FROM node:6-stretch AS build
 LABEL maintainer="will.price94@gmail.com"
 LABEL version="0.0.1"
-RUN apt-get update -qq && \
-    apt-get install -y python
 # Prevent npm from spamming
+ENV NPM_CONFIG_LOGLEVEL=warn
 RUN npm config set loglevel=warn && \
     npm config set progress=false
 WORKDIR /app/
 COPY package.json package-lock.json ./
 RUN npm install
 COPY . .
-RUN npm run build
+RUN mkdir -p /build/client
+RUN npm run build -- --outdir /build/client
+RUN mkdir -p /build/server
+RUN npm run buildserver -- --outdir /build/server
 
+FROM node:6-stretch
```

Comment:
I wonder whether we can get away with [`node:6-slim`](https://github.com/nodejs/docker-node/blob/17c50cb300581280805a4183524fbf57840f3a7e/6.11/slim/Dockerfile)

Reply:
Yes, yes we can! We save 750MB!

Reply:
We could possibly make it even smaller by using alpine, but I worry that the `sqlite` stuff won't work as it uses `node-gyp` and if we compile on debian then move over to alpine we may well see glibc/musl issues.

Reply:
Nope, not an issue, Woo hoo!